util/timer/src/timer.rs

```diff
@@ -34,11 +34,8 @@ pub trait TimeoutHandler: Send + Sync {
     fn on_timeout(&self, _token: TimerToken) {}
 }
 
-type TimeoutHandlerMap = HashMap<TimerId, Arc<TimeoutHandler>>;
```

Comment:
Would you explain what was the reason that you used `TimeoutHandlerMap`?

Reply:
I was intended it to be used for removing `TimeoutHandler`, but it wasn't realized.